source/Handlebars/IO/HtmlEncoder.cs

```diff
@@ -35,17 +35,18 @@ public void Encode(string text, TextWriter target)
             
             EncodeImpl(text, target);
         }
-        
-        private static void EncodeImpl<T>(T text, TextWriter target) where T: IEnumerator<char>
+
+        private void EncodeImpl<T>(T text, TextWriter target) where T : IEnumerator<char>
```

Comment:
Why changing it from `static` to instance method?

Reply:
Fixed. No reason, this was left from my first implementation where it did need to be instance method.